### PR TITLE
For #118: added non_final_attribute pattern

### PR DESCRIPTION
**Pull request by paulodamaso**
For #118: added non_final_attribute pattern and tests

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#134](https://github.com/yegor256/aibolit/issues/134) is assigned to @fanifieiev[/z](https://www.0crat.com/u/fanifieiev), here is [why](https://www.0crat.com/footprint/CT5UB7XAP/17d9684d-35cf-4916-9041-6e6d090b4b6e); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @acheshkov[/z](https://www.0crat.com/u/acheshkov) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: aibolit/patterns/non_final_attribute/non_final_attribute.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        :return: Tree
+        """
+        with open(filename, encoding='utf-8') as file:
+            tree = javalang.parse.parse(file.read())
+        return tree
+
+    def value(self, filename: str):
+        lst = []
+        tree = self.__file_to_ast(filename).filter(javalang.tree.FieldDeclaration)
+        for path, node in tree:
```
Comment (fanifieiev) [inline aibolit/patterns/non_final_attribute/non_final_attribute.py:22]: @paulodamaso I think it is better to:
`return [node for node, _ in tree if 'final' not in node.modifiers]`

File: aibolit/patterns/non_final_attribute/non_final_attribute.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import javalang
+
+
+class NonFinalAttribute:
+
+    def __init__(self):
+        pass
+
+    def __file_to_ast(self, filename: str) -> javalang.ast.Node:
```
Comment (fanifieiev) [inline aibolit/patterns/non_final_attribute/non_final_attribute.py:9]: @paulodamaso This one is copy-pasted over the project, so I created a ticket for that https://github.com/yegor256/aibolit/issues/131

**Comment by paulodamaso** [comment]
@fanifieiev Done, please take a look

**Comment by fanifieiev** [comment]
> @fanifieiev Done, please take a look

@paulodamaso Looks better, thx

**Comment by fanifieiev** [comment]
@acheshkov This PR is good to be merged

**Comment by acheshkov** [comment]
@rultor merge

**Pull request merged**